### PR TITLE
Fsxac/proposal3

### PR DESCRIPTION
**Pull request by FSXAC**
- Changed pronouns
- Added more terms and abbreviations

### CODE REVIEW

File: docs/proposal/1_1_business_background.tex [hunk 1]
```diff
@@ -1,3 +1,3 @@
-The client of this project is Professor Mieszko Lis, a professor in the Electrical and Computer Engineering department at the University of British Columbia (UBC). A particular research focus of Dr. Lis is the acceleration of (traditionally software-based) machine-learning (ML) models through the use of dedicated hardware. 
+The client of this project is Professor Mieszko Lis, a professor in the Electrical and Computer Engineering department at the University of British Columbia (UBC). One of the client's particular research focus is the acceleration of (traditionally software-based) machine-learning (ML) models through the use of dedicated hardware.
```
Comment (FSXAC) [inline docs/proposal/1_1_business_background.tex]:
```suggestion
The client of this project is Professor Mieszko Lis, a professor in the Electrical and Computer Engineering department at the University of British Columbia (UBC). One of the client's particular research foci is the acceleration of (traditionally software-based) machine learning (ML) models through the use of dedicated hardware.
```

File: docs/proposal/terms_and_abbreviations.tex [hunk 2]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
     \textbf{Term} & \textbf{Definition}\\
     \hline
 
-    NN &\\
-    CNN &\\
-    RNN &\\
-    YOLO &\\
-    TX & Transmitter.\\
+    ANN & Artificial Neural Network, or simply ``Neural Network'', is a data processing model modeled after neuron interactions. The process consists of forward propagation using several matrix multiplications.\cite{ann}\\
+    ASIC & Application-specific Integrated Circuits.\\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]:
```suggestion
    ASIC & Application-Specific Integrated Circuits.\\
```

File: docs/proposal/terms_and_abbreviations.tex [hunk 3]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
+    CNN & Convolutional neural network are neural networks that is especially useful for image classification.\cite{cnn} \\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]:
```suggestion
    CNN & Convolutional neural networks are neural networks which are especially useful for image classification.\cite{cnn} \\
```

File: docs/proposal/terms_and_abbreviations.tex [hunk 4]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
+    ECE & Department of Electrical and Computer Engineering at the University of British Columbia.\\
+    FPGA & Field-Programmable-Gate-Arrays, a ``programmable'' hardware that allows ASIC-like performance with software-like turn-around time and flexibility.\\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]:
```suggestion
    FPGA & Field-Programmable-Gate-Arrays, ``programmable'' hardware that allows ASIC-like performance with software-like turn-around time and flexibility.\\
```

File: docs/proposal/terms_and_abbreviations.tex [hunk 5]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
+    GPU & Graphics Processing Unit, a discrete piece of hardware designed to accelerate graphic-intensive or other parallel computing tasks.\\
+    LOS & Line-of-sight.\\
+    ML & Machine learning.\\
+    Multirotor & An unmanned vehicle with multiple engines. \\
+    OTS & Off-the-shelf, or commercially available/purchaseable \\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]:
```suggestion
    OTS & Off-the-shelf, or commercially available/purchasable \\
```

File: docs/proposal/terms_and_abbreviations.tex [hunk 6]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
+    PID / PID Controller & Proportion-Integral-Derivative controllers is the most common control algorithm for precise and accurate movement, as well as to compensate external forces.\cite{pid}\\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]: Are these an algorithm or a device?

File: docs/proposal/terms_and_abbreviations.tex [hunk 7]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
+    RNN & Recurrent neural networks are neural networks where the output depends on previous computations, essentially consists of memory.\cite{rnn}\\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]:
```suggestion
    RNN & Recurrent neural networks are neural networks where the output depends on previous computations, effectively emulating memory.\cite{rnn}\\
```

File: docs/proposal/terms_and_abbreviations.tex [hunk 8]
```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
...
     RX & Receiver.\\
     TC & Transport Canada.\\
+    TX & Transmitter.\\
+    YOLO & You-Only-Look-Once is a fast algorithm that detect objects but is unlike CNN nor RNN.\cite{yolo}\cite{yolo-2}\\
```
Comment (peterdeutsch) [inline docs/proposal/terms_and_abbreviations.tex]:
```suggestion
    YOLO & You-Only-Look-Once is a fast ML algorithm that detect objects but is unlike CNN nor RNN.\cite{yolo}\cite{yolo-2}\\
```

File: docs/proposal/terms_and_abbreviations.tex (hunk 6 above)
Comment (FSXAC) [reply docs/proposal/terms_and_abbreviations.tex]: I would say it's more of an algorithm. Not a software specific algorithm, but a general algorithm

File: docs/proposal/terms_and_abbreviations.tex (hunk 7 above)
Comment (FSXAC) [reply docs/proposal/terms_and_abbreviations.tex]: Hmm. I don't know if it's accurate to say it's emulating memory, which suggests that RNN is emulating human memory.

**Pull request merged**